README.md

```diff
@@ -25,7 +32,9 @@ Pointless Feedback has been tested against Rails 3.2+. To install, simply add
 the following to your Gemfile:
 
 ```ruby
-gem 'pointless_feedback', :git => 'git@github.com:vigetlabs/pointless-feedback.git'
+gem 'pointless_feedback',
+  :github => 'vigetlabs/pointless-feedback',
+  :branch => 'rails-4'
```

Comment:
Yes.  This is going to be merged and cut (hopefully).  I'll fix this in the docs.